aws-batch-processing-hillshade/batch/script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+cd ./app
+tmp_dir=$(mktemp -d -t hs-XXXXXXXXXX)
```

Comment:
generally use long form arguments to make it easier to read `-d` vs `--directory`